dataEmbedder.py

```diff
@@ -132,8 +138,23 @@ def encrypt_data(self, data: bytes) -> bytes:
         对要嵌入的数据用self.key加密。
         *yzy*
         """
-        # 可以用Crypto库里的现有算法。
-        pass
+        aes = AES.new(self.key, AES.MODE_ECB)
+        data = data.decode('utf8')
+        while len(data) % 16!=0:
+            data += '\0'
+        ciphertext = aes.encrypt(data.encode('utf8'))
```

Comment:
优化，直接对bytes进行填充；而且while效率太低！
```suggestion
        if len(data) % 16 != 0:
            # 填充至16字节的倍数
            data += b'\0' * (16 - len(data) % 16)
        ciphertext = aes.encrypt(data)
```